src/main/resources/application.yml

```diff
@@ -36,6 +36,7 @@ feign:
         loggerLevel: full
 folio:
   system-user:
+    enabled: ${SYSTEM_USER_ENABLED:true}
     username: ${SYSTEM_USER_NAME:system-user}
```

Comment:
AFAIK, we are not creating system user in this module. The reason we kept system user related properties is to avoid DI errors from folio-spring-system-user library.